.ci/cico_rhche_prcheck.sh

```diff
@@ -37,7 +37,7 @@ echo "Installing all dependencies lasted $instal_dep_duration seconds."
 
 export PROJECT_NAMESPACE=prcheck-${RH_PULL_REQUEST_ID}
 export DOCKER_IMAGE_TAG="${RH_TAG_DIST_SUFFIX}"-"${RH_PULL_REQUEST_ID}"
-CHE_VERSION=$(curl -s https://raw.githubusercontent.com/redhat-developer/rh-che/master/pom.xml | xq -r '.project.parent.version')
+CHE_VERSION=$(getVersionFromPom)
```

Comment:
how does `getVersionFromPom` work?

Reply:
the method is being sourced from `.ci/functional_tests_util.sh`
it's calling
```
scl enable rh-maven33 "mvn -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn org.apache.maven.plugins:maven-help-plugin:evaluate -q -Dexpression=project.parent.version -DforceStdout"
```
to get the version from the pom in the currently checked out branch of the project.

Reply:
If `yq` is installed, then `xq -r '.project.parent.version' pom.xml` could be a replacement as well.